channeldb/payment_status.go

```diff
@@ -4,10 +4,6 @@ package channeldb
 type PaymentStatus byte
 
 const (
-	// StatusUnknown is the status where a payment has never been initiated
-	// and hence is unknown.
-	StatusUnknown PaymentStatus = 0
```

Comment:
Would add a note explaining that the 0 value was deprecated here (and not shifted up to remain aligned with lnrpc values) - somebody else may be tempted to change it.

Reply:
Done.